### ISSUE TITLE
RGB color option

### ISSUE THREAD

**Issue by bogzbonny** [msokalski/ascii-patrol#30]
WOW! (incredible work) It would be sweet to have a 3rd option of enabling terminal RGB color-mode instead of 16-color (or black and white). Anyone who has a terminal with custom 16 color mappings has the color of this game altered too. (In my terminal my bright-white and white were nearly the same as I like a high contrast however I had to make white more dim in order to play this game). 

Just a thought!

**Comment by msokalski** [comment]
Yup, I think I'd add cmdline arg for using RGB esc codes instead of color indexes and just leave 16c / mono in game option as is.
